naptime/src/main/pegasus/org/coursera/naptime/schema/ReverseRelationAnnotation.courier

```diff
@@ -20,4 +20,12 @@ record ReverseRelationAnnotation {
    */
   relationType: enum RelationType { FINDER, MULTI_GET, GET, SINGLE_ELEMENT_FINDER }
 
+  /**
+   * This is an optional field which will override the auth headers from the
```

Comment:
let's fix up this comment to not refer to headers. indicating that a different auth strategy from the default behavior I think is fine.